mappings/net/minecraft/test/TestContext.mapping

```diff
@@ -45,8 +56,12 @@ CLASS net/minecraft/unmapped/C_ncfucati net/minecraft/test/TestContext
 		ARG 1 entity
 		ARG 2 pos
 		ARG 3 speed
-	METHOD m_exkyxbtk addFinalTask (Ljava/lang/Runnable;)V
-		ARG 1 runnable
+	METHOD m_eqeskuqo useBlock (Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_jzrpycqo;Lnet/minecraft/unmapped/C_jdakttms;)V
+		ARG 2 player
+		ARG 3 hit
+	METHOD m_exkyxbtk succeedIf (Ljava/lang/Runnable;)V
+		COMMENT Completes the test if {@code assertion} runs without throwing an exception, fails otherwise
```

Comment:
```suggestion
		COMMENT Completes the test if the provided {@code assertion} runs without throwing an exception, fails otherwise
```
little clarification, again